Xamarin.Forms.Platform.Android/AppCompat/MasterDetailContainer.cs

```diff
@@ -160,5 +107,59 @@ public void SetFragmentManager(FragmentManager fragmentManager)
 			if (_fragmentManager == null)
 				_fragmentManager = fragmentManager;
 		}
+
+		void SetCurrentFragment(Fragment fragment)
+		{
+			FragmentTransaction fragmentTransaction = FragmentManager.BeginTransaction();
+
+			if (_currentFragment != null)
+				fragmentTransaction.Remove(_currentFragment);
+
+			fragmentTransaction.Add(Id, fragment);
+			fragmentTransaction.SetTransition((int)FragmentTransit.None);
+
+			// The view we're hosting in the fragment was never created (possibly we're already 		
+			// navigating to another page?) so there's nothing to commit synchronously.
```

Comment:
I added the word `synchronously` to the comment. I believe that's what was meant since we're not executing pending transactions?